examples/submit_transaction/submit_tx.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/bitcoin-sv/go-broadcast-client/broadcast"
+	broadcast_client "github.com/bitcoin-sv/go-broadcast-client/broadcast/broadcast-client"
+)
+
+func main() {
+	token := ""
+	apiURL := "https://tapi.taal.com/arc"
+	tx := broadcast.Transaction{RawTx: "0100000001d6d1607b208b30c0a3fe21d563569c4d2a0f913604b4c5054fe267da6be324ab220000006b4830450221009a965dcd5d42983090a63cfd761038ff8adcea621c46a68a205f326292a95383022061b8d858f366c69f3ebd30a60ccafe36faca4e242ac3d2edd3bf63b669bcf23b4121034e871e147aa4a3e2f1665eaf76cf9264d089b6a91702af92bd6ce33bac84a765ffffffff0123020000000000001976a914d8819a7197d3e221e15f4348203fdecfd29fa2b888ac00000000"}
```

Comment:
Just a question:
Doesn't it contain some private things inside, in other words: didn't you put some private data  that transaction?

Reply:
Nope, another random tx